content/References/IPv6.md

```diff
@@ -0,0 +1,247 @@
+  This combines with the network prefix received from the router advertisement message that includes network configuration information.
+
+  DHCPv6, which is DHCP with the v6 extension, is an alternative that can still be used in IPv6 networks for more advanced configuration options, such as DNS server assignments and additional parameters.
+{{< /expand >}}
+
+IPv6 adds new capabilities:
+* {{< expand "Built-in auto-configuration" "v" >}}
+
+  IPv6 supports stateful and stateless auto-configuration (SLAAC).
+  Stateful auto-configuration requires some human intervention because it needs a DHCPv6 server for the installation and administration of nodes.
+  The DHCPv6 server maintains state information so the server knows how long each address is in use and when it might be available for reassignment.
+
+  Stateless auto-configuration (SLAAC) is suitable for small organizations and individuals.
+  It can automatically configure IPv6 host parameters on an IPv6 host without the need for manual configuration or a DHCP server.
+  Each host determines its addresses from the contents of received router advertisements and using the IEE EUI-64 standard to define the network ID portion of the address, to assume uniqueness of the host address on the link. For more information see [IPv6 Stateless Address Auto-configuration (SLAAC)](https://www.networkacademy.io/ccna/ipv6/stateless-address-autoconfiguration-slaac).
+{{< /expand >}}
+
+* {{< expand "Improved multicast routing" "v" >}}
+
+  IPv6 multicast routing uses multicast groups that logically-group devices that are interested in receiving the multicast traffic.
+  These groups are associated with a specific multicast address, which is typically in the range of 244.0.0.0 to 239.255.255.255 (IPv4) or the IPv6 range starting with the prefix ff00::/8. 
+
+  Routers in an IPv6 network must be configured to support multicast routing and enable an IPv6 multicast routing protocol like Protocol Independent Multicast (PIM) for IPv6.
+  Routers in the network use the multicast routing protocol to determine how to forward multicast packets based on multicast group membership.
+  Packet forwarding occurs to interfaces where devices have joined the corresponding multicast group interested in receiving the traffic.
+  With IPv6, multicast boundaries can constrain traffic within specific boundaries or domains of the network to control propagation.
+  {{< /expand >}}
+
+* {{< expand "Simpler header format" "v" >}}
+
+  IPv6 headers omit the IPv4 flags and fragmentation offset, time to live (TTL), Internet header length, header checksum, and protocol bits in routing headers.
+  It uses a flow label for identification, traffic class that is similar to the IPv4 type of service portion of the header, and adds a hop limit, payload length, and next header.
+  Both IPv6 and IPv4 include the source and destination in the header but allocates fewer bits to IPv4 headers (32 bits) than what is in IPv6 (128 bits).
```

Comment:
Wording here is a little confusing in terms of subject/object. What allocates fewer bits?